### PR TITLE
Add passport document type to Picoleto

### LINKED ISSUE #2: Detect and validate Passport

### PR DESCRIPTION
**Pull request by alorma**
Closes #2

### CODE REVIEW

File: src/test/java/NIFValidatorTest.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import com.fewlaps.picoleto.validators.NIFValidator;
+import  com.fewlaps.picoleto.validators.NIFValidator;
```
Comment (EsteveAguilera) [inline src/test/java/NIFValidatorTest.java]: Can you undo this change?

File: src/test/java/PassportValidatorTest.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    @Test
+    public void shouldReturnFalse_forANullPassport() {
+        String validPassport = null;
+
+        assertFalse(PassportValidator.withDocument(validPassport).isValid());
+    }
+
+    @Test
+    public void shouldReturnTrue_forAnEmptyPassport() {
```
Comment (EsteveAguilera) [inline src/test/java/PassportValidatorTest.java]: The test name is worng :wink:

Comment (alorma) [reply src/test/java/PassportValidatorTest.java]: Ooops

File: src/test/java/NIFValidatorTest.java (hunk 1 above)
Comment (alorma) [reply src/test/java/NIFValidatorTest.java]: Yes i can

**Comment by alorma** [comment]
Changes done @EsteveAguilera

**Comment by EsteveAguilera** [comment]
Gerat job @alorma! Thank you! :clap:  :+1:
